_guides/2021-09-30-data-maturity-open-source.md

```diff
@@ -18,33 +18,36 @@ outgoing: false
 
 ## What is Data Maturity?
 
-Data maturity is a concept used to create structure around an organization’s data capabilities, providing organized frameworks that guide their work. Data maturity encompasses multiple dimensions. For example, certain frameworks seek to characterize an organization’s data infrastructures, as well as the activities, skills, knowledge, leadership buy-in, ethics, and more. There are several frameworks that conceptualize these different elements, albeit with different foci, which serve as models for social impact organizations toward becoming more data mature (see Table 1).
+Data maturity is a concept used to create structure around an organization’s data capabilities, providing organized frameworks that guide their work. Data maturity encompasses multiple dimensions. For example, certain frameworks seek to characterize an organization’s data infrastructures, as well as the activities, skills, knowledge, leadership buy-in, ethics, and more. There are several frameworks that conceptualize these different elements, albeit with different foci, which serve as models for social impact organizations toward becoming more data mature (see [Table 1](#table-1-common-data-maturity-frameworks-in-the-social-sector)).
+
+### Table 1: Common Data Maturity Frameworks in the Social Sector
 
-**Selection of Two Common Data Maturity Frameworks in The Social Sector**
 
 | Data Maturity Framework | Summary |
 | ----------------------- | ------- |
-| [Data Orchard/DataKind UK’s Data Maturity Framework](https://drive.google.com/file/d/1wA_QKt49xzdxBguqQiFMu6VBgzEnCyZP/view) | Data Orchard’s and DataKind UK’s framework examines Data Maturity through seven lenses: leadership, skills, culture, data, tools, uses, and analysis. |
-| [CARE’s Responsible Data Maturity Model](https://www.ictworks.org/wp-content/uploads/2019/11/Responsible_Data_Maturity_Model_10-16-19.pdf) | CARE’s model provides a planning tool for organizations to evaluate their current data practices to identify resources and support for future efforts. In addition to examining elements, such as data and technology readiness, CARE emphasizes the need for “responsible” action to respond to the ethical, legal, social, and privacy-related challenges that come from using data. |
+| <img src="/assets/img/posts/v1-data-orchard-data-kind-mf.png" alt="Data Kind-Data Orchard maturity framework" width="250"> | [Data Orchard’s and DataKind UK’s framework](https://drive.google.com/file/d/1wA_QKt49xzdxBguqQiFMu6VBgzEnCyZP/view) examines data maturity through seven lenses: leadership, skills, culture, data, tools, uses, and analysis. |
```

Comment:
I'm using HTML here as it provides better flexibility when customizing the visuals (size, position, etc). I noticed that some other content can also use this approach. Example: https://merlcenter.org/guides/open-source-merl-workflows/